### PR TITLE
added a helpers file

### PR DESCRIPTION
**Pull request by Ccinaza**
I edited a line in the readme file and added a helper's file.

### CODE REVIEW

**Comment by ungest** [comment]
You have a good eye to notice the typo on in the readme file

**Review by Tekhunt — APPROVED** [review]
LGTM

**Review by Techkene — APPROVED** [review]
we understand you are testing this push

**Review by Afezhus — APPROVED** [review]
welldone

**Review by ungest — APPROVED** [review]

**Review by Ekeminimbabah — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ We are a passionate team of data enthusiasts who love to dive deep into data, an
 
 ## Projects
 
-Here are some of the projects we'reintereted in:
+Here are some of the projects we're interested in:
```
Comment (ungest) [inline readme.md]: Could you please change this to 'We are'

**Pull request merged**